ros/angel_msgs/msg/ObjectDetection3dSet.msg

```diff
@@ -22,6 +22,15 @@ int64 num_objects
 # Vector of detection axis aligned bounding box bounds point coordinates.
 # Each vector here should be of congruent length, where value `i` corresponds
 # to the 3D coordinate for detection `i`.
+#
```

Comment:
Now that I'm looking at this again, the 3D-space coordinate locations are not named great: there are no generic "sides" like there are in 2D pixel space. Your graphic is good to have, and I wouldn't complain if you updated the naming, but I guess that has ramifications that reach into the Unity app space.